src/integration/kotlin/pl/edu/agh/gem/integration/controller/ExpenseControllerIT.kt

```diff
@@ -348,5 +349,44 @@ class ExpenseControllerIT(
             // then
             response shouldHaveHttpStatus NOT_FOUND
         }
+
+        should("get expenses") {
+            // given
+            val groupMembers = GroupMembersResponse(listOf(GroupMemberResponse(USER_ID)))
+            stubGroupManagerMembers(groupMembers, GROUP_ID)
+            val expense = createExpense(expenseParticipants = listOf(createExpenseParticipant()))
+            expenseRepository.create(expense)
+
+            // when
+            val response = service.getGroupExpenses(createGemUser(USER_ID), GROUP_ID)
+
+            // then
+            response shouldHaveHttpStatus OK
+            response.shouldBody<GroupExpensesResponse> {
+                expenses shouldHaveSize 1
+                expenses.first().also {
```

Comment:
you could directly check expanse expenses.fist() shoulb be expense

Reply:
i don't think so. expense object is of type Expense and expenses.first() is of type GroupExpensesResponse which has different fields